src/test/java/io/outfoxx/swiftpoet/test/FileSpecTests.kt

```diff
@@ -268,7 +268,7 @@ class FileSpecTests {
           
           class Test {
 
-            let a: Array
+            let a: Swift.Array
```

Comment:
This makes sense because above we do:
```
      .addProperty("a", typeName("Swift.Array"))
      .addProperty("b", typeName("Foundation.Array"))
```